tests/integration/cli/config_test.py

```diff
@@ -27,4 +27,4 @@ def test_config_set_fails_on_nonexistent_key(self):
     def test_config_set_works_with_valid_input(self):
         self.run_command('config set nginx_includes_dir /var/nginx')
         result = yaml.load(self.run_command('config listvalues'))
-        self.assertEqual(result['nginx_includes_dir'], '/var/nginx')
+        self.assertEqual(result['nginx_includes_dir'], u'/var/nginx')
```

Comment:
Yeah, and with it.  The issue is that json loads makes all the strings unicode, even if you dumped non-unicode strings.  Apparently using yaml.safe_load gets around this (but won't work the same for weird character sets)